sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/SessionRecords/RestorableDatabaseAccountTests/RestorableDatabaseAccountListAsync.json

```diff
@@ -623,106 +623,107 @@
+            "enablePartitionKeyMonitor": false,
+            "isVirtualNetworkFilterEnabled": true,
+            "virtualNetworkRules": [],
+            "EnabledApiTypes": "Sql",
+            "disableKeyBasedMetadataWriteAccess": false,
+            "enableFreeTier": false,
+            "enableAnalyticalStorage": false,
+            "analyticalStorageConfiguration": {
+                "schemaType": "WellDefined"
             },
-            "secondaryMasterKey": {
-              "generationTime": "2023-04-21T10:21:32.5023247Z"
+            "instanceId": "b8cc042d-c5ff-43c4-94b5-d79eb5d1428b",
+            "createMode": "Default",
+            "databaseAccountOfferType": "Standard",
+            "enableMaterializedViews": false,
+            "defaultIdentity": "FirstPartyIdentity",
+            "networkAclBypass": "None",
+            "disableLocalAuth": false,
+            "enablePartitionMerge": false,
+            "enableBurstCapacity": false,
+            "minimalTlsVersion": "Tls",
+            "consistencyPolicy": {
+                "defaultConsistencyLevel": "BoundedStaleness",
+                "maxIntervalInSeconds": 1000,
+                "maxStalenessPrefix": 300
             },
-            "primaryReadonlyMasterKey": {
-              "generationTime": "2023-04-21T10:21:32.5023247Z"
+            "configurationOverrides": {},
+            "writeLocations": [
+                {
+                    "id": "r-database-account-6799-westus",
+                    "locationName": "West US",
+                    "documentEndpoint": "https://r-database-account-6799-westus.documents.azure.com:443/",
+                    "provisioningState": "Succeeded",
+                    "failoverPriority": 0,
+                    "isZoneRedundant": false
+                }
+            ],
+            "readLocations": [
+                {
+                    "id": "r-database-account-6799-westus",
+                    "locationName": "West US",
+                    "documentEndpoint": "https://r-database-account-6799-westus.documents.azure.com:443/",
+                    "provisioningState": "Succeeded",
+                    "failoverPriority": 0,
+                    "isZoneRedundant": false
+                }
+            ],
+            "locations": [
+                {
+                    "id": "r-database-account-6799-westus",
+                    "locationName": "West US",
+                    "documentEndpoint": "https://r-database-account-6799-westus.documents.azure.com:443/",
+                    "provisioningState": "Succeeded",
+                    "failoverPriority": 0,
+                    "isZoneRedundant": false
+                }
+            ],
+            "failoverPolicies": [
+                {
+                    "id": "r-database-account-6799-westus",
+                    "locationName": "West US",
+                    "failoverPriority": 0
+                }
+            ],
+            "keyVaultKeyUri": "",
```

Comment:
We can't intentionally create an instance of `URI` with empty string, by adding this line to service response to reproduce the issue.